rest/rest_collection_impl.h

```diff
@@ -561,7 +561,8 @@ addBackgroundJobInThread(Key key,
 
         // Set up the task, without starting it yet
         auto task = std::make_shared<BackgroundTask>();
-        task->config = std::move(config);
+        task->running = false;
```

Comment:
Shouldn't we set it to false at construction?  I am surprised that this was not the case.